yaml/service.go

```diff
@@ -157,11 +151,8 @@ func (s *ServiceSlice) Validate(pipeline []byte) error {
 					return err
 				}
 
-				invalid = fmt.Errorf(
-					"%w: %s",
-					invalid,
-					fmt.Errorf("invalid image value %s:\n%s\n ", service.Image, string(source)),
-				)
+				// nolint:cSpell // ignore line length
+				invalid = fmt.Errorf("%w: %s", invalid, fmt.Errorf("invalid image value %s:\n%s\n ", service.Image, string(source)))
```

Comment:
Wrong linter

```suggestion
				// nolint:lll // ignore line length
				invalid = fmt.Errorf("%w: %s", invalid, fmt.Errorf("invalid image value %s:\n%s\n ", service.Image, string(source)))
```

Reply:
Oh weird, I didn't even notice these were getting flagged because I was trying to stick to only modifying the secrets files per the title. I can clean these up though

Reply:
This we got it: https://github.com/go-vela/types/pull/126/commits/5e73730a830b92981cbed927ec09b1301111efac 😄